pkg/scheduler/scheduler.go

```diff
@@ -315,26 +308,24 @@ func (m *Scheduler) processAllocationUpdateEvent(ev *schedulerevent.SchedulerAll
         return
     }
 
-    // Allocations events cannot contain accepted and rejected allocations at the same time.
-    // There should never be more than one accepted allocation in the list
-    // See cluster_info.processAllocationProposalEvent()
     if len(ev.AcceptedAllocations) > 0 {
-        alloc := ev.AcceptedAllocations[0]
-        // decrease the outstanding allocation resources
-        m.clusterSchedulingContext.updateSchedulingNodeAlloc(alloc)
+        for _, alloc := range ev.AcceptedAllocations {
+            // decrease the outstanding allocation resources
```

Comment:
There is no change on the event generation side so this change does not make sense, the event can still only contain 1 accepted allocation.